.gitignore

```diff
@@ -2,9 +2,9 @@
 .vscode
 .vs
 # Don't check in any compiled results
-x64
-Build
-packages
+vs/Build/x64
```

Comment:
If you exclude all of `vs/Build` is this line necessary?

Reply:
I don't know. the `x64` was there, which was excluding `Build/x64` as well as `vs/Build/x64`. I don't think `x64` was ever needed, but it was there for some reason. Any idea what that reason is?

Reply:
I think `x64` used to be our build directory (historical from when the solution was a single project), I think we now have a more complicated build directory structure that is excluded by the next line.